docs/Migrating_LLB_to_a_PPL_based_Custom_Device.md

```diff
@@ -0,0 +1,128 @@
+## Migrating from LLB to PPL
+
+First introduced with LabVIEW 2010, a packed project library (PPL) is a compiled LabVIEW project library (.lvlib) that contains all the .project library VIs and allows the user to call the public VIs in a manner that is identical to the use of the original .project library. Since LabVIEW 2017 you can compile a packed project library to be also used with newer versions of LabVIEW. This option is enabled by default, but if you want to change that setting, choose "Advanced" in the sidebar and disable the option "Allow future versions of LabVIEW" to load this packed library. 
```

Comment:
```suggestion
First introduced with LabVIEW 2010, a packed project library (PPL) is a compiled LabVIEW project library (.lvlib) that contains all the .project library VIs and allows the user to call the public VIs in a manner that is identical to the use of the original project library. Since LabVIEW 2017 you can compile a packed project library to be also used with newer versions of LabVIEW. This option is enabled by default, but if you want to change that setting, choose "Advanced" in the sidebar and disable the option "Allow future versions of LabVIEW" to load this packed library. 
```